region/crateria/central/Flyway.json

```diff
@@ -379,7 +379,9 @@
         "2) With a momentum-conserving turnaround through the transition, or",
         "3) Jumping specifically with a extra run speed $5.2 or $5.3 and aiming down through the transition.",
         "For the first two methods, avoid backing into the corner at the start of runway;",
-        "instead press against it and turn around, to put Samus into a better position."
+        "instead press against it and turn around, to put Samus into a better position.",
+        "An even better position is obtained by backing up an additional 8 to 12 pixels from the runway start,",
```

Comment:
Does this mean back into the corner and then do 4-6 arm pumps?

Reply:
Still pressing forward into the corner, not backing into it. Added another line to clarify